packages/core/saved-objects/core-saved-objects-base-server-internal/src/constants.ts

```diff
@@ -114,3 +114,126 @@ export const DEFAULT_INDEX_TYPES_MAP: IndexTypesMap = {
     'workplace_search_telemetry',
   ],
 };
+
+/**
+ * In order to be FIPS compliant, the migration logic has switched
+ * from using hashes (stored in _meta.migrationMappingPropertyHashes)
+ * to using model versions (stored in _meta.mappingVersions).
+ *
+ * This map holds a breakdown of md5 hashes to model versions.
+ * This allows keeping track of changes in mappings for the different SO types:
+ * When upgrading from a Kibana version prior to the introduction of model versions for V2,
+ * the V2 logic will map stored hashes to their corresponding model versions.
+ * These model versions will then be compared against the ones defined in the typeRegistry,
+ * in order to determine which types' mappings have changed.
+ */
+export const HASH_TO_VERSION_MAP = {
```

Comment:
This will have to be updated "just before" the PR is merged, right? (or is it just before the version is cut?)

Reply:
We have an integration test to ensure it stays up to date

Reply:
Perfect then